### ISSUE TITLE
Fix color contrast issues on data explorer

### ISSUE THREAD

**Issue by toolness** [18F/calc#1553]
When I added aXe to our test suite in #1552, I disabled the color contrast check because it brought up a number of violations on the data explorer.  I think most of the violations were extremely minor, e.g. a contrast ratio of 4.5:1 was required but ours was 4.45:1.

Anyways, it'd be good to get them out of the way and re-enable the contrast check.  Or we can just wait until we refresh the data explorer w/ the new color scheme that @ericronne made.

**Comment by ericronne** [comment]
How do i view the violations?

**Comment by afeld** [comment]
Some tools that can be used for this: https://accessibility.18f.gov/tools/

**Comment by hbillings** [comment]
Closing because we have the new color schemes!
